config/database.example.yml

```diff
@@ -1,32 +0,0 @@
-# SQLite version 3.x
```

Comment:
This file should not be deleted(or renamed ), Please remove this from this PR

Reply:
and that's reason why test fail too
![image](https://user-images.githubusercontent.com/23432278/73667538-b399bf80-46a4-11ea-93c6-a005d2447ac0.png)